unity-ggjj/Assets/Scripts/BGScenes/BGSceneList.cs

```diff
@@ -1,21 +1,51 @@
 using System.Collections.Generic;
+using System.Linq;
 using UnityEngine;
 
 public class BGSceneList : MonoBehaviour
 {
-    private Dictionary<string, BGScene> _scenesInChildren;
-
+    [Tooltip("Drag a NarrativeScriptPlaylist here")]
+    [SerializeField] private NarrativeScriptPlaylist _narrativeScriptPlaylist;
+    
+    private readonly Dictionary<string, BGScene> _bgScenes = new Dictionary<string, BGScene>();
     private BGScene _activeScene;
 
     /// <summary>
-    /// Initializes the background scene dictionary
+    /// Gets all available BGScenes and instantiates them.
+    /// </summary>
+    public void Start()
```

Comment:
Nit: I believe Unity lifecycle methods aren't required to be `public` and as re-calling them is probably not a good idea, I'd suggest making it `private`.
```suggestion
    private void Start()
```

Reply:
Whoops haha.